meetup/models.py

```diff
@@ -1,29 +1,24 @@
 from django.db import models
 from pizzaplace.models import PizzaPlace
 from django.core.validators import RegexValidator
-from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from meetup.services.meetup_api import MeetupApi
 from django.core.exceptions import ValidationError
-
+from model_utils.models import TimeStampedModel
 
 def validate_urlname(link):
   validator = RegexValidator(
     regex='meetup\.com\/\w+(-\w+)*\/$',
-    message="Does not conform to Meetup Url",
+    message="Url should be in form 'meetup.com/meetup-name/'",
```

Comment:
I'd prefer to paste this in as a full URL, including the protocol: `http://meetup.com/meetup-name`.